### DISCUSSION TITLE
Home

### DISCUSSION THREAD

**Discussion by rusty-snake** [netblue30/firejail#4441] [Wiki]
Discussion for discussions about https://github.com/netblue30/firejail/wiki/Home.

Predecessor: #2749

**Comment by kmk3** [comment]
@rusty-snake commented [on Mar 24](https://github.com/netblue30/firejail/issues/2749#issuecomment-805720202):

> You can not create a PR for a GitHub Wiki ¹. The description is wrong and
> should be updated. For minor changes like typos just edit the page. For major
> changes it's better to discuss them in a issue first.
> 
> ¹ since a GitHub Wiki is at the end a git-repo with
> markdown/wiki-systax/reStructuredText/... files, it's possible to have a
> github.com/netblue30/firejail-wiki repo which is synced to the wiki. This
> repo would support PRs.

By the way, we did exactly this on a project that I worked on, also because of
wanting to discuss changes on PRs beforehand.  The main issue was that there
was no automation for it, so the syncing had to be done manually.  Thus, to
make things easier to maintain, the -wiki repo was made the master copy and the
actual wiki was effectively a mirror (IIRC). 

With this setup, the syncing is trivial to do, but it makes contributing a bit
more cumbersome (i.e.: having to edit pages on a separate place).

If the wiki is meant to still be directly editable, then it's a bit more
complex, as the syncing would have to work on both ways.

I think that can be automated with GH Actions.  In such case, maybe each sync
could work reasonably well by doing fast-forward merges and automatically
opening an issue (on the wiki repo) if conflicts are detected.  Though if both
syncs happen quickly enough and on every commit, maybe conflicts would be very
rare outside of PRs.

Anyway, I wish there was a way to just configure a normal separate repository
(e.g.: firejail-wiki) as being the actual wiki.  In which case, firejail/wiki
would show pages directly from the configured repository.
